joomla/joomla-cms
# [4.2] Remove the SCSS deprecations

Pull Request for Issue # .

### Summary of Changes

- Removing the math.div deprecations from the scss compiler 
- update Font Awesome to v6
- update choices.js to v10

### Testing Instructions
- Apply this Pr's branch
- run `npm ci`
- Check that nothing is broken visually


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required
